lib/fdonet.c

```diff
@@ -405,8 +422,9 @@ bool resolve_dn(const char *dn, fdo_ip_address_t **ip, uint16_t port,
 		}
 	}
 end:
-	if (ip_list) // free ip_list
+	if (ip_list) {// free ip_list
```

Comment:
nitpick - should there be a space between the brace and comment (or comment on separate line)